site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc

```diff
@@ -18,6 +18,23 @@ Your Webhook must handle the RESTful request described above and send back an ap
 
 Once your Webhook is complete and listening for events, you must configure your Webhook URL in FusionAuth. To add a webhook navigate to [breadcrumb]#Settings -> Webhooks#. If you have multiple Webhooks configured for a single Application, the transaction setting for the event or the User Action will dictate if FusionAuth will commit the transaction or not.
 
+=== Handling Create Events
```

Comment:
We could say the same for `user.registration.create` or `user.update`, or `user.registration.update`. Depending upon what you expect in your webhook, the final state of this transaction will not likely be complete during the execution of your webhook. 

To really support these use cases, we would need to fire post events that would never be transactional and would be fired after we commit the TX.

Reply:
Good call, I'll make it clearer that this is an issue no matter what, but more sticky when the user/other entity doesn't exist.